src/styles/_catalog.scss

```
@@ -1,9 +1,25 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (max-width: 1023px) {
+    & { 
+      grid-template-columns: repeat(3, 200px);
+    }
+  }
```

You dont need `&` here
```suggestion
  @media (max-width: 1023px) {
    grid-template-columns: repeat(3, 200px);
  }
```